webpack/assets/javascripts/react_app/components/ToastsList/__snapshots__/ToastsList.test.js.snap

```diff
@@ -1,714 +0,0 @@
-              onDismiss={[Function]}
-              onMouseEnter={[Function]}
-              onMouseLeave={[Function]}
-              type="success"
-            >
-              <div
-                className="alert toast-pf alert-success alert-dismissable"
-                onMouseEnter={[Function]}
-                onMouseLeave={[Function]}
-              >
-                <Button
-                  active={false}
-                  aria-hidden="true"
-                  block={false}
-                  bsClass="close"
-                  bsStyle="default"
-                  disabled={false}
-                  onClick={[Function]}
-                >
-                  <button
-                    aria-hidden="true"
-                    className="close close-default"
-                    disabled={false}
-                    onClick={[Function]}
-                    type="button"
-                  >
-                    <Icon
-                      name="close"
-                      type="pf"
-                    >
-                      <PatternflyIcon
-                        className=""
-                        name="close"
-                      >
-                        <span
-                          aria-hidden="true"
-                          className="pficon pficon-close"
-                        />
-                      </PatternflyIcon>
-                    </Icon>
-                  </button>
-                </Button>
-                <Icon
-                  name="ok"
-                  type="pf"
-                >
-                  <PatternflyIcon
-                    className=""
-                    name="ok"
-                  >
-                    <span
-                      aria-hidden="true"
-                      className="pficon pficon-ok"
-                    />
-                  </PatternflyIcon>
-                </Icon>
-                <AlertBody
-                  link={
-                    Object {
-                      "children": "hi link!",
-                      "href": "google.com",
-                    }
-                  }
-                  message="Widget positions successfully saved."
-                >
-                  <span>
-                    <AlertLink
-                      href="google.com"
-                    >
-                      <div
-                        className="pull-right toast-pf-action"
-                      >
-                        <a
-                          href="google.com"
-                        >
-                          hi link!
-                        </a>
-                      </div>
-                    </AlertLink>
-                    Widget positions successfully saved.
-                  </span>
-                </AlertBody>
-              </div>
-            </Alert>
-          </ToastNotification>
-        </TimedToastNotification>
-      </div>
-    </ToastNotificationList>
-  </ToastsList>
-</Connect(ToastsList)>
-`;
```

Comment:
wow! 714 lines are gone, nice!
we still need a small integration test and we're done :)